types/vote_test.go

```diff
@@ -53,6 +53,70 @@ func TestVoteSignable(t *testing.T) {
 	require.Equal(t, expected, signBytes, "Got unexpected sign bytes for Vote.")
 }
 
+func TestVoteSignableTestVectors(t *testing.T) {
+	voteWithVersion := CanonicalizeVote("", &Vote{Height: 1, Round: 1})
+	voteWithVersion.Version = 123
+
+	tests := []struct {
+		canonicalVote CanonicalVote
+		want          []byte
+	}{
+		{
+			CanonicalizeVote("", &Vote{}),
+			// NOTE: Height and Round are skipped here. This case needs to be considered while parsing.
+			[]byte{0xb, 0x2a, 0x9, 0x9, 0x0, 0x9, 0x6e, 0x88, 0xf1, 0xff, 0xff, 0xff},
```

Comment:
how do I get this something changes?

Reply:
It's just the output of `MarshalBinary`. The test serves two purposes: 1) mainly documentation for people wanting to parse this without having to fully implement amino/proto3 (like @jleni) 2) get notified if the encoding changes for whatever reason.